plugins/woocommerce/src/Blocks/Utils/ProductCollectionUtils.php

```diff
@@ -92,4 +99,174 @@ private static function remove_empty_array_recursive( $array ) {
 		}
 		return $array;
 	}
+
+	/**
+	 * Parse editor's location context from WP Post.
+	 *
+	 * Possible contexts:
+	 * - post
+	 * - page
+	 * - product
+	 * - product-archive
+	 * - cart
+	 * - checkout
+	 * - catalog
+	 * - order
```

Comment:
Nitpick, the comment is behind the code update:

```suggestion
	 * Possible contexts:
	 * - post
	 * - page
	 * - single-product
	 * - product-archive
	 * - cart
	 * - checkout
	 * - product-catalog
	 * - order-confirmation
```